src/transformStateWithClones.js

```diff
@@ -7,7 +7,55 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const statesArray = [];
+  const currentState = { ...state };
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
- You can use for..of cycle.
 for (let action of actions) {}
 - and you can use a destructuring assignment:
const { type, extraData, keysToRemove } = action;

Reply:
agree with [comment](https://github.com/mate-academy/js_stateful-clones/pull/2399/files#r1226370386) 
please try to rewrite the code by using `for ... of`